examples/dynamic_rels/main.rs

```diff
@@ -27,36 +27,36 @@ model:
        resolver: resolve_project_top_contributor
 ";
 
-fn resolve_project_top_contributor(facade: ResolverFacade<()>) -> ExecutionResult {
-    // create dynamic dst node
-    let mut top_contributor_props = HashMap::<String, Value>::new();
-    top_contributor_props.insert(
-        "id".to_string(),
-        Value::from(Uuid::new_v4().to_hyphenated().to_string()),
-    );
-    top_contributor_props.insert("name".to_string(), Value::from("user0".to_string()));
-    let top_contributor = facade.create_node("User", top_contributor_props);
+fn resolve_project_top_contributor(facade: ResolverFacade<'_, ()>) -> BoxFuture<ExecutionResult> {
```

Comment:
@maldrake : why is a lifetime needed in the ResolverFacade here but not in the same place in the `dynamic_props` example?

Reply:
Good catch. That lifetime marking isn't needed.  I removed it in the latest push.